actions/restore.py

```diff
@@ -190,11 +191,20 @@ def probe_forked_etcd():
 def reconfigure_client_advertise():
     ''' Reconfigure the backup to use host network addresses for client advertise
         instead of the assumed localhost addressing '''
-    cmd = "/snap/bin/etcd.etcdctl member list"
-    members = check_output(split(cmd))
-    member_id = members.split(b':')[0].decode('utf-8')
 
-    raw_update = "/snap/bin/etcd.etcdctl member update {0} http://{1}:{2}"
+    loop = 0
+    MAX_WAIT = 10
```

Comment:
`MAX_WAIT` really is "MAX_LOOPS`.  I don't see any "waiting" going on (ie -- sleeps or whatever).  Did you have an intent to wait?

Reply:
Seems like I've forgot to add `sleep`, I aim to retry the command a couple of times until etcd cluster settles. I will be fixing it.